### ISSUE TITLE
nanmin no longer works with series

### ISSUE THREAD

**Issue by rstoneback** [pandas-dev/pandas#8383]
Using NumPy's nanmin on a pandas Series worked as expected, now however the Series is returned with each element set to the minimum, rather than just getting the minimum. Not sure if this is a Numpy of Pandas issue.

```
np.nanmin(pds.Series([1,2,3,4]))
Out[14]: 
0    1
1    1
2    1
3    1
dtype: int64

np.min(pds.Series([1,2,3,4]))
Out[15]: 1
```

**Comment by topper-123** [comment]
This issue has been solved:

```python
>>> np.nanmin(pds.Series([1,2,3,4]))
1
```

I'm on numpy 1.13.1 and pandas 0.22.

**Comment by jreback** [comment]
yup. though its possibly only for certainly versions of numpy. @topper-123 can you add some tests and see?

**Comment by topper-123** [comment]
This is solved only for numpy >= 1.13. Where would tests for this be placed?

**Comment by jreback** [comment]
test_nanops
